src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/vsts_cd_provider.py

```diff
@@ -0,0 +1,81 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=no-self-use,too-many-arguments,too-many-lines
+from __future__ import print_function
+from sys import stderr
+from vsts_cd_manager.continuous_delivery_manager import ContinuousDeliveryManager
+import adal
+from azure.cli.core._profile import Profile, CredsCache
+
+
+class VstsContinuousDeliveryProvider(object):
+    def __init__(self):
+        self._progress_last_message = ''
+
+    def setup_continuous_delivery(self, resource_group_name, name, repo_url, branch, git_token,
+                                  slot, cd_app_type, cd_account, cd_create_account, location):
+        """
+        This method sets up CD for an Azure Web App thru Team Services
+        :param resource_group_name:
+        :param name:
+        :param repo_url:
+        :param branch:
+        :param git_token:
+        :param slot:
+        :param cd_app_type:
+        :param cd_account:
+        :param cd_create_account:
+        :param location:
+        :return:
+        """
```

Comment:
Please fill the docstring instead of leaving an empty skeleton. Otherwise please just leave a short description of the method.